pkg/client/unversioned/request.go

```diff
@@ -244,11 +246,10 @@ func (r *Request) AbsPath(segments ...string) *Request {
 	if r.err != nil {
 		return r
 	}
-	if len(segments) == 1 {
+	r.path = path.Join(r.proxyPrefix, path.Join(segments...))
```

Comment:
So with my preferred storage location, this would become `r.path = path.Join(r.baseURL.Path, segments...)`. Will anything go wrong with that?